pages/deployment/aws.mdx

```diff
@@ -0,0 +1,180 @@
+
+### Picking the OS 
+
+When creating the EC2 instance, you need to pick the OS you wish to have on the
+instance. If you are going to use the Memgraph native installation, you need to
+pick a Linux distribution that Memgraph supports.
+
+Memgraph supports multiple Linux distributions. Memgraph packages for supported distributions and versions can be downloaded 
+from the [following page](/getting-started/install-memgraph/direct-download-links#linux)
+
+
+This way, you will manage the native installation of Memgraph, which can bring
+some of the speed benefits. 
+
+If you are going to use the Memgraph Docker image, you can pick any Linux
+distribution you prefer, as long as you can run Docker on it.
+
+
+### Picking the instance type
+
+When creating the EC2 instance, you need to pick the instance type. The process
+highly depends on your actuall workload. For heavy graph processing, consider
+using instances with more CPU and memory. 
+
+If you are running Memgraph in `IN_MEMORY_TRANSACTIONAL` (default mode) or in `IN_MEMORY_ANALYTICAL` storage mode, all data is stored in RAM. That means it would be good to first calculate [how much memory you will
+need](/fundamentals/storage-memory-usage#calculate-storage-memory-usage).
+There is also an [easy-to-use
+calulator](https://memgraph.com/enterprise#ram-calculator) for approximating
+memory usage. The good rule of thumb is to have double the memory of what your
+storage would take. You can go with less if you do not have a demanding
+workload.
+
+
+AWS has specialized instances for [memory-optimized
+workloads](https://docs.aws.amazon.com/ec2/latest/instancetypes/mo.html) which
+are designed for in-memory databases, data analytics, and other memory-intensive
+applications. They also provide DDR5 memory, so you may consider using such instances to
+optimize for better performance. 
+
+
+### Setup the network
+
+When creating the EC2 instance, you need to set up the network access and
+security group. By default, Memgraph uses port 7687 for the Bolt protocol.
+You need to open this port for TCP traffic and to allow connections to Memgraph. 
+
+If you change the default Bolt port, make sure to update the security group
+accordingly.
+
+Also, if deploying Memgraph for [Replication](/clustering/replication#replication) or [High
+Availability](/clustering/high-availability#high-availability-enterprise), the ports for the
+replication and clustering should be open as well.
+
+
+### Setup the storage 
+
+When creating the EC2 instance, you need to set up the storage. By default
+Memgraph stores all data to working RAM, but for the [persistency between
+restarts](../fundamentals/data-durability) Memgraph uses the disk storage to
+store snapshots, configurations, etc.
+
+The sizing of the storage depends on the amount of data you are going to store
+in your Memgraph instance, and the quantity of the snapshots you want to keep
+alive. 
+
```

Comment:
Do we recommend EBS or local storage? Why? Does it matter if the storage is magnetic or SSD?

Reply:
Added a bit of color here.